src/interfaces/IPluginExecutor.sol

```diff
@@ -20,4 +20,14 @@ interface IPluginExecutor {
         external
         payable
         returns (bytes memory);
+
+    /// @notice Execute a call using a specified runtime validation, as given in the first 21 bytes of
+    /// `authorization`.
+    /// @param data The calldata to send to the account.
+    /// @param authorization The authorization data to use for the call. The first 21 bytes specifies which runtime
+    /// validation to use, and the rest is sent as a parameter to runtime validation.
+    function executeWithAuthorization(bytes calldata data, bytes calldata authorization)
```

Comment:
Do we also need to update https://github.com/erc6900/reference-implementation/blob/6d48a68021bff6f3a50a508873595af9fb1939a6/standard/ERCs/erc-6900.md in this PR or later on